webapp/channels/package.json

```diff
@@ -34,6 +34,7 @@
     "core-js": "3.26.0",
     "country-list": "2.2.0",
     "crypto-browserify": "3.12.0",
+    "crypto-js": "4.1.1",
```

Comment:
We already have a `crypto-browserify` library. Can you try to use that instead? Agreed, it might not have straight forward APIs for generating words array. Let me know if you need help with that. :)

Reply:
It's possible. However it will require some changes in the implementation.
At first, I had trouble importing the lib with usual react import. Typescript required `@types/crypto-browserify` and I keep getting 404 when I run `npm install --save-dev @types/crypto-browserify` . 
To bypass that, I imported it using nodejs require. Like this:
~~~ ts
const crypto = require('crypto');
~~~
Please let me know if you guys don't see a problem on importing like that or if I missed something in the regular import that got me those problems.

The behavior is exactly the same and the implementation changes to this:
~~~ ts
setURL(crypto.randomBytes(16).toString('hex'));
~~~

Reply:
You are absolutely correct we have aliased in `webpack.config.js`. There are other places in codebase which imports `crypto` as well. The above implementation looks good to me i checked out locally. Would you like to proceed with that

Reply:
Yes. I just pushed the commit with that change.